docs/src/__types__/index.ts

```diff
@@ -1,7 +1,14 @@
+export type SliderInputType = {
+    min: number;
+    max: number;
+    step: number;
+};
+
 export type PropsType = {
     propName: string;
     inputType: string;
-    inputValue: boolean | string | string[] | [];
+    inputValue: boolean | number | string | string[] | [];
+    inputSets?: SliderInputType;
```

Comment:
I don't understand why this variable is called `inputSets` when it contains rangeData for a slider.  Seems weird to me.

Reply:
> We also don't have any code highlight on the angular side; I don't think it's required because it's going to make things a lot more difficult further along.
> 
> On the angular side, if a prop has been reset to its default value, the prop no longer appears in the code snippet. This variance will cause the highlighted section to constantly be updated, so I'd just remove the highlighting all together. It's too difficult to maintain & too timely to test.

Remove line highlighting.

Reply:
Renamed inputSets  => rangeData